rasa/core/policies/policy.py

```diff
@@ -272,43 +295,6 @@ def _featurize_for_prediction(
             == SupportedData.ML_DATA,
         )
 
-    def train(
```

Comment:
I deleted them since there is no rule what a graph component receives as parameter during training or running (this only depends on the `needs` section)

Reply:
re-add